tock/utilization/unit.py

```diff
@@ -0,0 +1,60 @@
+from django.conf import settings
```

Comment:
Is this blocked on getting the units into a database? Pretty much all the current info will break when we do that, I expect.

Reply:
Yeah, we'll hold off on major work here until that issue is wrapped and merged. Shouldn't be too tough to accommodate